src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -43,7 +43,8 @@
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\amd64\git2-69db893.dll" target="lib\NativeBinaries\amd64" />
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\x86\git2-69db893.dll" target="lib\NativeBinaries\x86" />
     <file src="Microsoft.DotNet.Build.Tasks\AssemblyInfoPartial.*" target="lib" />
-    <file src="GenFacades\GenFacades.dll" target="lib" />
+	<file src="BclRewriter\BclRewriter.exe" target="lib" />
+    <file src="GenFacades\GenFacades.exe" target="lib" />
```

Comment:
@weshaggard note that I am changing Genfacades to be .exe instead of .dll so we will have to make a change in corefx repo if we are already consuming it